frontend/src/global_styles/content/_widget_box.sass

```diff
@@ -116,6 +116,10 @@ $widget-box--enumeration-width: 20px
   border-bottom: none
   padding-bottom: 0
 
+  .help-text--entry
+    vertical-align: 2px
+    padding-right: 5px
```

Comment:
Shouldn't we use the same spacing (`var(--base-size-6`/`6px`) as [for the new Primer implementation](https://github.com/opf/openproject/blob/dev/app/components/open_project/common/attribute_label_component.sass#L34)?